app/assets/stylesheets/container_providers_dashboard.css

```diff
@@ -0,0 +1,92 @@
+    padding-top: 125px;
+}
+
+/* line 10, scss/main.scss */
+.miq-body {
+    background: #f5f5f5;
+}
+
+/* line 16, scss/main.scss */
+.miq-dashboard-view .card-pf-aggregate-status .card-pf-title .pficon, .miq-dashboard-view .card-pf-aggregate-status .fa, .miq-objects-view .card-pf-aggregate-status .card-pf-title .pficon, .miq-objects-view .card-pf-aggregate-status .fa {
+    color: #1186C1 !important;
+}
+/* line 21, scss/main.scss */
+.miq-dashboard-view .data-list-pf .list-group-item.active .pficon, .miq-dashboard-view .data-list-pf .list-group-item.active:hover .pficon, .miq-dashboard-view .data-list-pf .list-group-item.active:focus .pficon, .miq-objects-view .data-list-pf .list-group-item.active .pficon, .miq-objects-view .data-list-pf .list-group-item.active:hover .pficon, .miq-objects-view .data-list-pf .list-group-item.active:focus .pficon {
+    color: #1186C1;
+}
+/* line 24, scss/main.scss */
+.miq-dashboard-view .data-list-pf .list-group-item .pficon, .miq-objects-view .data-list-pf .list-group-item .pficon {
+    font-size: 18px;
+}
+/* line 27, scss/main.scss */
+.miq-dashboard-view .tiles-view-pf .tile.active .pficon, .miq-dashboard-view .tiles-view-pf .tile.active:hover .pficon, .miq-dashboard-view .tiles-view-pf .tile.active:focus .pficon, .miq-objects-view .tiles-view-pf .tile.active .pficon, .miq-objects-view .tiles-view-pf .tile.active:hover .pficon, .miq-objects-view .tiles-view-pf .tile.active:focus .pficon {
+    color: #1186C1;
+}
+/* line 31, scss/main.scss */
+.miq-dashboard-view .card-pf .loading, .miq-objects-view .card-pf .loading {
+    margin: 10px auto;
+}
+
+/* line 35, scss/main.scss */
+.miq-dashboard-view .pficon-kubernetes::before, .miq-objects-view .pficon-kubernetes::before {
+    color: #326de6;
+}
+/* line 39, scss/main.scss */
+.miq-dashboard-view .pficon-openshift::before, .miq-objects-view .pficon-openshift::before {
+    color: #da2430;
+}
+
+/* line 44, scss/main.scss */
+.provider-card {
+    min-height: 144px;
+}
+/* line 47, scss/main.scss */
+.provider-card .providers {
+    padding: 20px 0;
+}
+/* line 50, scss/main.scss */
+.provider-card .providers a {
+    color: #000000;
+    text-decoration: none;
+}
+/* line 55, scss/main.scss */
+.provider-card .providers a:hover {
+    color: #00618a;
+    text-decoration: underline;
+}
+/* line 59, scss/main.scss */
+.provider-card .providers .count {
+    margin: 0 5px;
+}
+/* line 63, scss/main.scss */
+.provider-card .providers .provider-icon {
+    font-size: 22px;
+    position: relative;
+    top: 5px;
+    padding-right: 5px;
+    opacity: .9;
+}
+/* line 70, scss/main.scss */
+.provider-card .providers .provider-icon:hover {
+    opacity: 1;
+}
+/* line 76, scss/main.scss */
+.provider-card .provider-icon-large {
+    font-size: 100px;
+    position: relative;
+    padding-bottom: 4px;
+    width: 100%;
+    text-align: center;
+}
+
+/* line 85, scss/main.scss */
+.providers-status {
+    padding-bottom: 9px !important;
+}
+
+.card-pf-title {
```

Comment:
`.card-pf-title`? I'd say yes, as it's not part of patternfly or angular-patternfly .. but if you have a reason to keep it like this..?